javascript/javascript1/week1/Houseypricey.js

```diff
@@ -0,0 +1,29 @@
+//peter house cost 
+var houseWidth = 8; 
+var houseDepth = 10; 
+var houseHeight = 10;
+var gardenSize = 100;
+var houseCost = 2500000;
+var volumeInMeters = houseWidth * houseDepth * houseHeight;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+if(housePrice > houseCost) {
+   console.log( "peter did paye too much");
+}
+else (housePrice < houseCost);{
+    console.log ("peter did paye less");
+}
+//Julia house cost
+
+var houseWidth = 5; 
+var houseDepth = 11; 
+var houseHeight = 8;
+var gardenSize = 70;
+var houseCost = 1000000;
+var volumeInMeters = houseWidth * houseDepth * houseHeight;
```

Comment:
No need to re-declare these variables. You can declare them at the first time using `let`, and here just assign a new value